lib/testing/src/lib/process-services-cloud/pages/start-process-cloud-component.page.ts

```diff
@@ -114,4 +114,12 @@ export class StartProcessCloudPage {
             }
         });
     }
+
+    selectOptionAtIndex(index) {
```

Comment:
I think we can have a more significant name here

Reply:
type information is missing
```suggestion
    selectOptionAtIndex(index: number) {
```

Reply:
can't use a name as we are referring ot a process definition with no name.